exosphere-shared/example-apps/crashing-service/runner/server

```diff
@@ -1,4 +1,3 @@
 #!/usr/bin/env bash
 
 echo 'running ...'
-read test
```

Comment:
This example app contains two services: crasher and runner. Crasher crashes, while runner keeps on running. This file is the implementation of the runner service. Without this line, the service will also stop. It should keep running